webview/src/experiments/components/table/styles.module.scss

```diff
@@ -626,57 +629,21 @@ $badge-size: 0.85rem;
   font-size: 0.65rem;
 }
 
-.bullet {
-  visibility: visible;
-  align-items: center;
-  background: $checkbox-background;
-  border-radius: $bullet-size;
-  border: calc(var(--border-width) * 1px) solid $checkbox-border;
-  color: $icon-color;
+.plotBox {
+  height: $checkbox-size;
```

Comment:
And I now see that I'm basically just describing what Studio does in their design 😅 

<img width="140" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/074afaf2-4d3c-4171-adad-f5a6f4e93253">

Reply:
What color did you use for the selected svg fill in that screenshot?

Reply:
Updated to this: 

<img width="1728" alt="image" src="https://github.com/iterative/vscode-dvc/assets/37993418/9ba48673-9dec-45bf-8eeb-66bc2606e0a2">

WDYT?

Reply:
I just used white since it works as a foreground for all of our unchanging custom colors. Something like`var(--vscode-list-activeSelectionForeground)` works for most cases if we wanted to use a variable though!

Reply:
> Updated to this:
> WDYT?

Looks better to me!

Reply:
I went with `fill: 'var(--vscode-editor-foreground)'` for the selected svgs we can revisit later if needed.